### PR TITLE
send mapping and artist relation import request to spark cluster

### PR DESCRIPTION
**Pull request by vansika**
Rn, there is no way to send a request to import mapping and artist relation into the cluster. This PR resolves the issue.

### CODE REVIEW

**Comment by vansika** [comment]
again I have not tested this fully on my local machine. Can be tested on the cluster.

File: listenbrainz/spark/handlers.py [hunk 1]
```diff
@@ -203,3 +203,38 @@ def handle_recommendations(data):
                                                                similar_artist_recording_mbids)
 
     current_app.logger.debug("recommendation for {} inserted".format(musicbrainz_id))
+
+
+def handle_mapping(data):
```
Comment (mayhem) [inline listenbrainz/spark/handlers.py:208]: this name does not reflect what the function does

File: listenbrainz/spark/handlers.py [hunk 2]
```diff
@@ -203,3 +203,38 @@ def handle_recommendations(data):
...
+    send_mail(
+        subject='MSID MBID mapping has been imported into the Spark cluster',
+        text=render_template('emails/mapping_import_notification.txt', mapping_name=mapping_name, time=import_completion_time),
+        recipients=['listenbrainz-observability@metabrainz.org'],
+        from_name='ListenBrainz',
+        from_addr='noreply@'+current_app.config['MAIL_FROM_DOMAIN'],
+    )
+
+
+def handle_artist_relation(data):
```
Comment (mayhem) [inline listenbrainz/spark/handlers.py:225]: same here

File: listenbrainz/spark/handlers.py (hunk 1 above)
Comment (vansika) [reply listenbrainz/spark/handlers.py:208]: handle_mapping_imported?

Comment (mayhem) [reply listenbrainz/spark/handlers.py:208]: Your doc string clearly names the key function of this method. Why is that not part of the name?

Comment (vansika) [reply listenbrainz/spark/handlers.py:208]: `notify_mapping_import` ? 

Otherwise you tell me one :p

Comment (mayhem) [reply listenbrainz/spark/handlers.py:208]: Yep, that works!

**Comment by vansika** [comment]
@paramsingh ready for merge

**Pull request merged**